packages/wrangler/src/api/startDevWorker/devtools.ts

```diff
@@ -0,0 +1,41 @@
+import type Protocol from "devtools-protocol/types/protocol-mapping";
```

Comment:
The types from devtools-protocol don't expose the full object we see in the websocket messages, just the sub-properties for some reason

This file may seem daunting but it just coerces the types into the whole object we see – e.g. with `method` and `params` matched together not as separate type unions which is basically useless

@mrbbot got nerd-sniped and got it working 👏